src/main/java/org/jboss/tools/intellij/openshift/actions/cluster/SetCurrentClusterAction.java

```diff
@@ -11,7 +11,9 @@
 package org.jboss.tools.intellij.openshift.actions.cluster;
 
 import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.diagnostic.Logger;
```

Comment:
Use raw SLF4J API to be consistent and more standard

Reply:
done.